### PR TITLE
Added error handling for CSRF

### PR DESCRIPTION
**Pull request by nawazish587**
Added error handling for CSRF

Resolves: SEC-115

### CODE REVIEW

File: src/middleware/csrfErrorMiddleware.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ErrorRequestHandler, NextFunction, Request, Response } from 'express'
+import {
+    CsrfError
+} from '@companieshouse/web-security-node'
+
+// TODO: Enter the template name here instead of <TEMPLATE NAME>
+const csrfErrorTemplateName = "403";
```
Comment (SteveHicksCH) [inline src/middleware/csrfErrorMiddleware.ts:7]: Consider a more meaningful template name. These are just for devs so how about `csrf-error`

**Review by SteveHicksCH — COMMENTED** [review]
As well as comment in the code, you need to add a test to make sure that this page is accessed  Sonar Quality fails due to lack of tests for the changes

File: src/middleware/csrfErrorMiddleware.ts (hunk 1 above)
Comment (nawazish587) [reply src/middleware/csrfErrorMiddleware.ts]: thanks @SteveHicksCH I have modified to be consistent with existing middleware files.

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Sonar analysis show code smells on the import of ErrorRequestHandler not being used and the TODO comment should be removed.
Please remove the TODO comment on line 6 as it has been done and check if it works without the ErrorRequestHandler import.

**Review by hwilliams1 — APPROVED** [review]
Looks ok to me

**Pull request merged**